apps/rmh-web/app/pt-portal/page.tsx

```diff
@@ -15,9 +17,9 @@ export default function PTPortal() {
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <TopBar />
```

Comment:
this should probably go in the layout component at the top level since it will likely be used everywhere